src/lib-components/BlockCardWithImage.vue

```diff
@@ -152,6 +165,9 @@ const parsedDateFormat = computed(() => {
   </li>
 </template>
 
-<style lang="scss" scoped>
+<style
+  lang="scss"
+  scoped
+>
 @import "@/styles/themes.scss";
```

Comment:
it might be worth refactoring this import to just import the component specific files like sectionteasercard.vue is doing. This will prevent bloat in the components and might help with some style clashing in cardmeta and blockcardwithimage

Reply:
We decided to make a bug ticket for this because the change affects all the styles.